code/main.cpp

```diff
@@ -0,0 +1,54 @@
+#include "../headers/mp3_player.hpp"
```

Comment:
no need to include with relative paths #include "mp3_player.hpp" or <mp3_player.hpp>  is sufficient